src/main/cpp/hw/ctreadapters/pro/DragonPercentOutputToCTREProAdapter.cpp

```diff
@@ -25,22 +25,35 @@
 #include "mechanisms/controllers/ControlModes.h"
 
 // Third Party Includes
+#include "units/dimensionless.h"
+#include "units/voltage.h"
 
 using ctre::phoenixpro::controls::DutyCycleOut;
+using ctre::phoenixpro::controls::VoltageOut;
 using ctre::phoenixpro::hardware::TalonFX;
 using std::string;
 
 DragonPercentOutputToCTREProAdapter::DragonPercentOutputToCTREProAdapter(string networkTableName,
                                                                          int controllerSlot,
                                                                          const ControlData &controlInfo,
                                                                          const DistanceAngleCalcStruc &calcStruc,
-                                                                         DragonTalonFX &controller) : DragonControlToCTREProAdapter(networkTableName, controllerSlot, controlInfo, calcStruc, controller)
+                                                                         ctre::phoenixpro::hardware::TalonFX &controller) : DragonControlToCTREProAdapter(networkTableName, controllerSlot, controlInfo, calcStruc, controller)
 {
 }
 void DragonPercentOutputToCTREProAdapter::Set(double value)
 {
-    DutyCycleOut out{value};
-    m_controller->Set(out);
+    if (m_isDuty)
+    {
+        DutyCycleOut out{value};
+        out.WithEnableFOC(m_enableFOC);
+        m_controller.SetControl(out);
+    }
+    else if (m_isVoltage)
+    {
+        VoltageOut out{units::volt_t(value)};
```

Comment:
Does this need to take into account the maxVoltage?  From my understanding this would send a control for 1.0 volt at 100%.

Reply:
I believe this control mode expects actual voltage from the documentation.  Could be wrong.  The doco states that this is: Voltage to attempt to drive at